mobile/node.go

```diff
@@ -64,8 +64,30 @@ type Node struct {
 	ipfsConfig *ipfscore.BuildCfg
 	apiConfig  *repo.APIConfig
 }
+type Mobile struct { }
 
-func NewNode(config NodeConfig) (*Node, error) {
+
+func NewOB(repoPath string, authenticationToken string, testnet bool, userAgent string, walletTrustedPeer string, password string, mnemonic string) *Node {
```

Comment:
Objective-C does not like allocating structure objects to send into these bridge methods so I had to create a standard proxy function.